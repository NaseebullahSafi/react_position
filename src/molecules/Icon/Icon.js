import React from 'react'
import './Icon.scss'

const Icon = ({ children, ...rest }) => (
    <svg
        xmlns="http://www.w3.org/2000/svg"
        width="24"
        height="24"
        viewBox="0 0 24 24"
        fill="none"
        stroke="#4d5961"
        strokeWidth="2"
        strokeLinecap="round"
        strokeLinejoin="round"
        {...rest}
    >
        {children}
    </svg>
)

export default Icon