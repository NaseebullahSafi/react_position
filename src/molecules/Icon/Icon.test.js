import React from 'react';
import { render, cleanup } from '@testing-library/react';
import Icon from './Icon';

afterEach(cleanup)

describe('Icon Component', () => {
  test('Matches the snapshot', () => {
    const { asFragment } = render(<Icon />);
    expect(asFragment()).toMatchSnapshot()
  })
})