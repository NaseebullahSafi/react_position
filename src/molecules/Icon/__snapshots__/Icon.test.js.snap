// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon Component Matches the snapshot 1`] = `
<DocumentFragment>
  <svg
    fill="none"
    height="24"
    stroke="#4d5961"
    stroke-linecap="round"
    stroke-linejoin="round"
    stroke-width="2"
    viewBox="0 0 24 24"
    width="24"
    xmlns="http://www.w3.org/2000/svg"
  />
</DocumentFragment>
`;
